### PR TITLE
prettier config added

### PR DESCRIPTION
**Pull request by Zhenyatta**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -21,8 +20,8 @@ app.get("/", (req, res) => {
   ++pageVisitedTimesTotal;
```
Comment (GagMirz) [inline backend/index.js]: **praise:** don't use pre increment whenever it's not necessary action (From logic perspective), it's less readable than postincrement.

File: df [hunk 2]
```diff
@@ -0,0 +1,59 @@
+[33mcommit 3cb21144022e4b30428eb02567fa78eaae6bab84[m[33m ([m[1;36mHEAD -> [m[1;32mtest[m[33m, [m[1;31morigin/test[m[33m)[m
```
Comment (GagMirz) [inline df]: **praise:** Why have you pushed diff file?

File: backend/constants.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const MY_COUNTER_VISITED_TIMES_FILE_PATH = 'temp/my-counter-visited-times.txt'
```
Comment (GagMirz) [inline backend/constants.js]: **praise:** MY_COUNTER_VISITED_TIMES_FILE_PATH change to something shorter

**Review by RobMin — APPROVED** [review]
LGTM

**Pull request merged**